src/core/ext/client_config/lb_policy.h

```diff
@@ -127,9 +125,9 @@ void grpc_lb_policy_init(grpc_lb_policy *policy,
 /** Given initial metadata in \a initial_metadata, find an appropriate
     target for this rpc, and 'return' it by calling \a on_complete after setting
     \a target.
-    Picking can be asynchronous. Any IO should be done under \a pollent. */
+    Picking can be asynchronous. Any IO should be done under the pollset_set
+    interested_parties. */
```

Comment:
Thanks a lot for the review!
I was trying to say "... under a \a grpc_pollset_set called \a interested_parties". But "... under the \a interested_parties \a grpc_pollset_set in the \a grpc_lb_policy struct" seems better .